azure-pipelines.yml

```
@@ -74,9 +74,9 @@ stages:
           steps:
             - checkout: self
             - task: NuGetToolInstaller@1
-              displayName: 'Use NuGet 5.7.0'
+              displayName: 'Use NuGet 5.11.5'
```

Do we need to duplicate the version in the name? 

I don't need to read the version in every small PR I create under normal conditions.
In the logs, it's pretty clear what version will be used.
```
Downloading: https://dist.nuget.org/win-x86-commandline/v5.11.5/nuget.exe
Caching tool: NuGet 5.11.5 x64
Using version: 5.11.5
Found tool in cache: NuGet 5.11.5 x64
Using tool path: C:\hostedtoolcache\windows\NuGet\5.11.5\x64
Prepending PATH environment variable with directory: C:\hostedtoolcache\windows\NuGet\5.11.5\x64
```
I'd also rename it to install, as that's what the task `NuGetToolInstaller` does

```suggestion
              displayName: 'Install NuGet'
```

---

I removed the version altogether to align with sonar-dotnet and sonarlint repos.